examples/stable/test/end_to_end/candid_rpc/complex_init/src/complex_init/index.ts

```diff
@@ -13,7 +13,7 @@ export default class {
     init(): void {
         const tuple = IDL.decode(
             [IDL.Tuple(IDL.Text, User)],
-            msgArgData()
+            new Uint8Array(msgArgData()).buffer
```

Comment:
Show me the runtime example, this seems crazy

Reply:
Done